tests/monday_type_tests.py

```diff
@@ -144,4 +145,93 @@ def test_number_type_should_succeed_when_to_primitive_returns_export_string_when
 
     # Assert
     eq_(int_value,'1')
-    eq_(float_value,'1.0')
\ No newline at end of file
+    eq_(float_value,'1.0')
+
+
+def test_date_type_should_succeed_when_to_native_returns_a_datetime_when_passing_a_datevalue_value_with_api_data():
+
+    # Arrange
+    id = 'date_1'
+    title = 'Date'
+    column_type = ColumnType.date
+    date = datetime(2021, 9, 29, 17, 48, 35)
+    date = {'date': str(date.date()), 'time': str(date.time())}
+    value = json.dumps(date)
+    date_value = en.cv.create_column_value(column_type, id=id, title=title,value=value)
+
+    # Act
+    date_type = t.DateType(id='date_column_1')
+    value = date_type.to_native(date_value)
+
+    # Assert
+    eq_(value, date_value.value)
+
+
+def test_date_type_should_succeed_when_to_native_returns_a_datetime_when_passing_a_integer_or_string_value():
+
+    # Arrange
+    date_type = t.DateType(id='date_column_2', has_time=True)
+
+    # Act
+    value_1 = date_type.to_native(1632917915)
+    value_2 = date_type.to_native('2021-09-29 17:48:35')
+
+    # Assert
+    eq_(value_1, datetime(2021, 9, 29, 17, 48, 35))
+    eq_(value_2, datetime(2021, 9, 29, 17, 48, 35))
+
+
+def test_date_type_should_succeed_when_to_native_returns_none_when_passing_none():
+
+    # Arrange
+    date_type = t.DateType(id='date_column_3')
+
+    # Act
+    value = date_type.to_native(None)
+
+    # Assert
+    eq_(value, None)
+
+
+def test_date_type_should_succeed_when_to_primitive_returns_empty_dict_when_passing_none():
+
+    # Arrange
+    date_type = t.DateType(id='date_column_4')
+
+    # Act
+    value = date_type.to_primitive(None)
+
+    # Assert
+    eq_(value, {})
+
+
+def test_date_type_should_succeed_when_to_primitive_returns_export_dict_when_passing_datetime_value():
+
+    # Arrange
+    date_type = t.DateType(id='date_column_5')
+    value = datetime(2021, 9, 29)
+
+    # Act
+    value = date_type.to_primitive(value)
+
+    # Assert
+    eq_(value['date'], '2021-09-29')
+    eq_(value['time'], None)
+
+
+def test_date_type_should_succeed_when_to_primitive_returns_export_dict_with_time_when_passing_datetime_value_and_has_time_is_true():
+
+    # Arrange
+    date_type = t.DateType(id='date_column_6', has_time=True)
+    value = datetime(2021, 9, 29, 17, 48, 35)
+
+    # Act
+    value = date_type.to_primitive(value)
+
+    # Assert
+    eq_(value['date'], '2021-09-29')
+    eq_(value['time'], '17:48:35')
+
```

Comment:
The requirements documentation has also been updated with two new tests to add to account for the invalid date and time str formats.  Please add these in.